src/sentry_uuid.c

```diff
@@ -101,6 +102,27 @@ sentry_uuid_as_string(const sentry_uuid_t *uuid, char str[37])
 #undef B
 }
 
+void
+sentry__internal_uuid_as_string(const sentry_uuid_t *uuid, char str[37])
+{
+#define B(X) (unsigned char)uuid->bytes[X]
+    snprintf(str, 37,
```

Comment:
```suggestion
    snprintf(str, 33,
```
Since you are only serializing 16 hex bytes, and a zero byte.